test/Object/ObjectSpread.js

```diff
@@ -0,0 +1,219 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Object Spread unit tests
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+function verifyPropertyDesc(obj, prop, desc, propName) {
+    var actualDesc = Object.getOwnPropertyDescriptor(obj, prop);
+    if (typeof propName === "undefined") { propName = prop; }
+    assert.areEqual(desc.configurable, actualDesc.configurable, propName+"'s attribute: configurable");
+    assert.areEqual(desc.enumerable, actualDesc.enumerable, propName+"'s attribute: enumerable");
+    assert.areEqual(desc.writable, actualDesc.writable, propName+"'s attribute: writable");
+}
+
+var a = {i: 1, j: 2};
+var b = {x: 3, y: 4, z: 5};
+var c = {foo: 6};
+
+var tests = [
+    {
+        name: "Test shallow cloning",
+        body: function() {
+            let a = {i: 1, j: 2};

```

Comment:
Is it intentional to shadow the global scope `a` here?

Reply:
no, looks like I missed that